web_hosting_manager/CHANGELOG.md

```diff
@@ -3,7 +3,12 @@
 ## [Unreleased]
 ### Changed
 - Update to Node JS to ^8.0.0
-- Update @maidsafe/safe-node-app to v0.6.1 
+- Update @maidsafe/safe-node-app to v0.6.1
```

Comment:
Same comment as for the email app changelog. Also, please include the updated yarn.lock file for the WHM.